src/sentry/utils/prompts.py

```diff
@@ -42,3 +45,10 @@ def required_fields(self, name):
 
 
 prompt_config = PromptsConfig(DEFAULT_PROMPTS)
+
+
+@request_cache
+def check_prompt_activities(organization_ids, features):
```

Comment:
Should it just be called `get_prompt_activities`?

cc @wedamija for any thoughts on this

Reply:
Yeah, if it's returning them I'd call it `get/fetch`

Reply:
@EvanPurkhiser makes sense to me